pkg/blob/controllerserver.go

```diff
@@ -114,7 +114,14 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	}
 	if IsAzureStackCloud(d.cloud) {
 		accountKind = string(storage.Storage)
-		storageAccountType = "Standard_LRS"
+		if strings.HasPrefix(strings.ToLower(storageAccountType), "standard") && storageAccountType != string(storage.StandardLRS) {
+			klog.Warningf("Use sku %s instead as Azure Stack does not support %s.", storage.StandardLRS, storageAccountType)
+			storageAccountType = "Standard_LRS"
+		}
+		if strings.HasPrefix(strings.ToLower(storageAccountType), "premium") && storageAccountType != string(storage.PremiumLRS) {
+			klog.Warningf("Use sku %s instead as Azure Stack does not support %s.", storage.PremiumLRS, storageAccountType)
+			storageAccountType = "Premium_LRS"
```

Comment:
convert saliently may be not a good option, maybe add a supported account list and if it's not in the list, prompt error

Reply:
I agree. But I remember we had a discussion about this before -- for simplicity, internally convert the storage account type to a supported type, and mention that in the documentation is good enough for now. Otherwise, we also need to change the test cases for Azure Stack, which outweighs the benefits.